spec/controllers/reservations_controller_spec.rb

```diff
@@ -295,7 +246,7 @@
 
         context 'and user can override errors' do
           before(:each) do
-            allow(@ac).to receive(:override_on_create).and_return(true)
+            mock_app_config(AC_DEFAULTS.merge(override_on_create: true))
```

Comment:
So this overrides the first mock app config that we create in the highest level `before(:each)` call?